hlink/scripts/lib/conf_validations.py

```diff
@@ -279,10 +291,14 @@ def check_column_mappings(config, df_a, df_b):
                     raise ValueError(
                         f"Within a [[column_mappings]] the column_name: '{column_name}' does not exist in datasource_b and no previous [[column_mapping]] alias exists for it. Column mapping: {c}. Available columns: \n {df_b.columns}"
                     )
+        if column_name in columns_available:
+            duplicates.append(column_name)
```

Comment:
Note that we may have an alias that renames the column here, so we need to account for that. The alias is optional for column_mappings.

Reply:
Does the change I just made fix that?